### PR TITLE
Extend GSoC 2022 landing and documentation

### PR DESCRIPTION
**Pull request by oleg-nenashev**
Signed-off-by: Oleg Nenashev <o.v.nenashev@gmail.com>

### CODE REVIEW

File: mentorship/gsoc/README.md [hunk 1]
```diff
@@ -58,3 +68,11 @@ To add a new project idea, submit a pull requests against [this page](./2022/pro
...
 quick start guidelines and, if possible,
 references to specific newbie-friendly issues.
+
+## How does GSoC work in Keptn?
+
+During the organization application phase,
+we defined the top level framework about the GSoC framework in the Keptn organization.
+See [this document](https://docs.google.com/document/d/1MV7Y-_7vwgWfKwjDUemk_OOhQSt7qrp9RG9LSAQGGSE/edit?usp=sharing) for the details
+we submitted to Google during the application.
+Feel free toi ask on the `#gsoc` Slack channel if you have any questions.
```
Comment (mehabhalodiya) [inline mentorship/gsoc/README.md:78]:
```suggestion
Feel free to ask on the `#gsoc` Slack channel if you have any questions.
```

**Review by mehabhalodiya — COMMENTED** [review]
Just a little typo 🙂

**Comment by mehabhalodiya** [comment]
It looks good to me!

**Comment by oleg-nenashev** [comment]
Thanks for catching it @mehabhalodiya !

**Comment by Ciggzy1312** [comment]
Looks pretty good!

**Pull request merged**